taskcluster/app_services_taskgraph/transforms/toolchain.py

```diff
@@ -37,7 +37,9 @@ def resolve_keys(config, tasks):
         })
         # TODO: Bug 1637695 - temp solution to unblock local building of
         # appplication-services. Once we switch to new indexes, we should clean this up
-        sha = git_sha_for_directory("libs")
-        routes = task['routes']
-        routes.append(TOOLCHAIN_OLD_INDEX[task['name']].format(sha=sha))
+        if task['name'] in TOOLCHAIN_OLD_INDEX.keys():
```

Comment:
Note: these will be cleanup-ed soon, once we're 100% certain that we can retire the old (non-taskgraph) way of doing appsv releases. Until then, adding this check so that we don't break if we add a different toolchain task.